bin/cli.js

```diff
@@ -78,8 +78,9 @@ For more information, see https://webpack.js.org/api/cli/.`);
 				if (moduleName === "webpack") {
 					errorMessage = `\n${moduleName} not installed`;
 					instructions = `Install webpack to start bundling: \u001b[32m\n  $ npm install --save-dev ${moduleName}\n`;
-
-					if (process.env.npm_execpath !== undefined && process.env.npm_execpath.includes("yarn")) {
+					const path = require("path");
+					const fs = require("fs");
+					if (fs.existsSync(path.resolve(process.cwd(), "yarn.lock"))) {
```

Comment:
Have you tested this against yarn plug n play?

Reply:
No !!! , never used yarn pnp. will do it then

Reply:
Nice

Reply:
This is not 100% fool-proof either. I think the better is use both the checks

Reply:
yeah, we can add both too !

Reply:
Yeah let's go for that

Reply:
done already in the last commit  !!! 👍